android-kotlin/app/src/main/java/com/mongodb/app/MyApp.kt

```diff
@@ -0,0 +1,39 @@
+package com.mongodb.app
+
+import android.app.Application
+import android.util.Log
+
+import io.realm.Realm
+import io.realm.log.LogLevel
+import io.realm.log.RealmLog
+import io.realm.mongodb.App
+import io.realm.mongodb.AppConfiguration
+
+lateinit var myApp: App
+
+// global Kotlin extension that resolves to the short version
+// of the name of the current class. Used for labelling logs.
+inline fun <reified T> T.TAG(): String = T::class.java.simpleName
+const val PARTITION_EXTRA_KEY = "PARTITION"
+const val PROJECT_NAME_EXTRA_KEY = "PROJECT NAME"
```

Comment:
You can probably delete both of these lines.

Reply:
done